jetty-core/jetty-http/src/main/java/org/eclipse/jetty/http/HttpField.java

```diff
@@ -362,6 +362,56 @@ else if (c != '0' && c != '.')
         return param != ZERO_QUALITY.length() && match == search.length();
     }
 
+    /**
+     * Look for a value as the last value in a possible multivalued field
+     * Parameters and specifically quality parameters are not considered.
+     * @param search Values to search for (case-insensitive)
+     * @return True iff the value is contained in the field value entirely or
+     * as the last element of a quoted comma separated list.
+     */
+    public boolean containsLast(String search)
+    {
+        return containsLast(getValue(), search);
+    }
+
+    /**
+     * Look for the last value in a possible multivalued field
+     * Parameters and specifically quality parameters are not considered.
+     * @param value The field value to search in.
+     * @param search Values to search for (case-insensitive)
+     * @return True iff the value is contained in the field value entirely or
+     * as the last element of a quoted comma separated list.
+     */
+    public static boolean containsLast(String value, String search)
```

Comment:
I was just following the pattern used by the contains method of having a static so that if somebody does their own header value calculations the method can be used.  We don't do it, but I like consistency....   I'll inline if you really don't like it.

Reply:
I don't like the other statics either. I'm -0: prefer it to be inlined, but I can live with it.

Reply:
fixed in previous